src/Chapter2.hs

```diff
@@ -327,7 +327,10 @@ from it!
 ghci> :l src/Chapter2.hs
 -}
 subList :: Int -> Int -> [a] -> [a]
-subList = error "subList: Not implemented!"
+subList p1 p2 xs 
+   | p2 < p1 = []
+   | otherwise = let rest = drop p1 xs 
+                       in take (p2 - p1 + 1) rest
```

Comment:
Great one!
But it could work not as expected on negative input. You can check that both numbers are not negative 🙂